### PR TITLE
[C-634] Add Lightbox for profile page profile and cover photos

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description

Add the lightbox for the profile page profile and cover photos. Not too much logic here, but some repetition for checking if the photos exist so open to refactor things.

Will get feedback from Julian and Forrest before pushing anything

### How Has This Been Tested?

Manually tested, photo, no photo, and photos of different sizes appear to be working

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Add-lightbox-for-profile-page

File: packages/web/src/components/cover-photo/CoverPhoto.tsx [hunk 1]
```diff
@@ -41,13 +42,18 @@ const CoverPhoto = ({
...
   darken = false,
+  onClick = () => {},
   onDrop
 }: CoverPhotoProps) => {
   const [processing, setProcessing] = useState(false)
   const gradient = darken
     ? 'linear-gradient(180deg, rgba(0,0,0,0.25) 0%, rgba(0,0,0,0.75) 100%)'
     : 'linear-gradient(rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.05) 70%, rgba(0, 0, 0, 0.2) 100%)'
 
+  const hasCoverPhoto = Boolean(
```
Comment (sliptype) [inline packages/web/src/components/cover-photo/CoverPhoto.tsx:53]: Feel like there is a better way to do this, `hasProfilePicture` is computed like this:

```ts
    const hasProfilePicture = profile
      ? !!profile.profile_picture ||
        !!profile.profile_picture_sizes ||
        updatedProfilePicture
      : false
```
Wonder if something similar would work?

**Review by sliptype — APPROVED** [review]
🔥🔥🔥

This is sick

File: packages/web/src/components/cover-photo/CoverPhoto.tsx [hunk 2]
```diff
@@ -41,13 +42,18 @@ const CoverPhoto = ({
   error,
   edit = false,
   darken = false,
+  onClick = () => {},
```
Comment (dylanjeffers) [inline packages/web/src/components/cover-photo/CoverPhoto.tsx:45]: i think no need for default value here, can just do `onClick?.()`?

File: packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx [hunk 3]
```diff
@@ -611,6 +621,38 @@ const ProfilePage = ({
     elements
   })
 
+  const hasCoverPhoto = useMemo(() => {
+    return Boolean(
+      coverPhotoSizes?.[WidthSizes.SIZE_2000] ||
+        coverPhotoSizes?.[WidthSizes.SIZE_640]
+    )
+  }, [coverPhotoSizes])
+  const lightBoxImgUrl = useMemo(() => {
```
Comment (dylanjeffers) [inline packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx:630]: spacing

File: packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx [hunk 4]
```diff
@@ -611,6 +621,38 @@ const ProfilePage = ({
...
+        ''
+      )
+    return (
+      profilePictureSizes?.[SquareSizes.SIZE_480_BY_480] ??
+      profilePictureSizes?.[SquareSizes.SIZE_150_BY_150] ??
+      ''
+    )
+  }, [coverPhotoSizes, profilePictureSizes, lightboxImg])
+
+  const handleCoverPhotoClick = useCallback(() => {
```
Comment (dylanjeffers) [inline packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx:644]: nit, i think `handleClickCoverPhoto` what you think?

**Review by dylanjeffers — APPROVED** [review]
awesome, mind adding a before/after photo/vid?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Add-lightbox-for-profile-page

**Comment by sliptype** [comment]
Closing and will revisit as an actual project

**Pull request closed without merging**